plugin/registereditor.lua

```diff
@@ -1,9 +1,136 @@
+        end,
+    })
+
+    -- update open RegisterEdit buffers when text is yanked into a register
+    vim.api.nvim_create_autocmd({ "TextYankPost" }, {
+        group = autocommand_group,
+        callback = function()
+            local event = vim.api.nvim_get_vvar("event")
+            internals.update_register_buffers(
+                -- if no register was specified for the yank, then we will be
+                -- yanking into the " register
+                event.regname == "" and '"' or event.regname,
+                event.regcontents
+            )
+
+            -- update numbered registers
+            for register_number = 1, 10 do
+                local register = tostring(register_number - 1)
+                internals.update_register_buffers(
+                    register,
+                    vim.fn.getreg(register):split("\n")
+                )
+            end
+        end,
+    })
+
+    -- update open RegisterEdit buffers after using the command line
+    vim.api.nvim_create_autocmd({ "CmdlineLeave" }, {
+        group = autocommand_group,
+        callback = vim.schedule_wrap(function()
+            internals.refresh_all_register_buffers()
+        end),
+    })
+
+    -- update open RegisterEdit buffers for the - register. Vim has many ways
+    -- for the - register to update internally. Each event in this list needs
+    -- at least one example as a justification for being included
+    -- * TextChanged - needed when using x from normal mode
+    -- * InsertEnter - needed when using s from visual mode
+    vim.api.nvim_create_autocmd({ "TextChanged", "InsertEnter" }, {
+        group = autocommand_group,
+        callback = vim.schedule_wrap(function()
+            internals.update_register_buffers(
+                "-",
+                vim.fn.getreg("-"):split("\n")
+            )
+        end),
+    })
+
+    -- update open RegisterEdit buffers for the . register.
+    vim.api.nvim_create_autocmd({ "InsertLeave" }, {
+        group = autocommand_group,
+        callback = function()
+            internals.update_register_buffers(
+                ".",
+                vim.fn.getreg("."):split("\n")
+            )
+        end,
+    })
+
+    -- update open RegisterEdit buffers for the # and % registers.
+    vim.api.nvim_create_autocmd({ "WinEnter", "BufEnter" }, {
+        group = autocommand_group,
+        callback = function()
+            internals.update_register_buffers(
+                "#",
+                vim.fn.getreg("#"):split("\n")
+            )
+            internals.update_register_buffers(
+                "%",
+                vim.fn.getreg("%"):split("\n")
+            )
+        end,
+    })
+end
+
+local function setup_keymaps()
+    local update_slash_register = vim.schedule_wrap(function()
+        internals.update_register_buffers("/", vim.fn.getreg("/"):split("\n"))
+    end)
+    add_key_trigger("n", "*", update_slash_register)
+    add_key_trigger("n", "#", update_slash_register)
```

Comment:
There's also `*` and `#` in visual mode, probably simple to add

Reply:
Added.

Reply:
Problem: no longer does builtin command, see https://github.com/tuurep/registereditor/pull/12#issuecomment-3036953950